Date: Fri,  1 Jun 2018 17:00:29 +0200
From: Tomas Henzl <thenzl@redhat.com>
To: rhkernel-list@redhat.com
Subject: [RHEL7.6 e-stor 53/60] scsi: mpt3sas: Increase event log buffer to
 support 24 port HBA's.

For 24 port HBA's events generated by IOC are more in certain cases and
the current circular buffer may be overwritten.Hence increased the event
log buffer to accommodate more events.

Signed-off-by: Chaitra P B <chaitra.basappa@broadcom.com>
Signed-off-by: Suganath Prabu S <suganath-prabu.subramani@broadcom.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
(cherry picked from commit 1537d1bfc58858f2e0a86c37977d92b05e0afa2e)
---
 drivers/scsi/mpt3sas/mpt3sas_ctl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: src/drivers/scsi/mpt3sas/mpt3sas_ctl.h
===================================================================
--- src.orig/drivers/scsi/mpt3sas/mpt3sas_ctl.h	2018-09-05 17:29:34.182174038 +0200
+++ src/drivers/scsi/mpt3sas/mpt3sas_ctl.h	2018-09-07 06:23:16.909306971 +0200
@@ -184,7 +184,7 @@
 
 
 /* number of event log entries */
-#define MPT3SAS_CTL_EVENT_LOG_SIZE (50)
+#define MPT3SAS_CTL_EVENT_LOG_SIZE (200)
 
 /**
  * struct mpt3_ioctl_eventquery - query event count and type
